.github/workflows/security.yml

```
@@ -0,0 +1,625 @@
+    branches: [main, develop]
+  schedule:
+    # Run security scans daily at 4 AM UTC
+    - cron: '0 4 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+  security-events: write
+    inputs:
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)